torchvision/prototype/transforms/_auto_augment.py

```diff
@@ -337,7 +337,7 @@ def __init__(
         magnitude: int = 9,
         num_magnitude_bins: int = 31,
         interpolation: InterpolationMode = InterpolationMode.NEAREST,
-        fill: Optional[Union[FillType, Dict[Type, FillType]]] = None,
+        fill: Union[FillType, Dict[Type, FillType]] = None,
```

Comment:
This type hint looks a bit strange, we expect to see optional somewhere but surprise FillType includes None and this is not obvious. OK with that as it is just a typehint

Reply:
True. But if we don't add `None` in `FillType`, we would constantly need to do `Union[Optional[FillType], Dict[Type, Optional[FillType]]` which is very verbose. Would you prefer this idiom?

Reply:
I totally understand your poinrt and I'm OK with None inside FillType.